allennlp/models/archival.py

```diff
@@ -132,6 +144,12 @@ def archive_model(
         archive.add(weights_file, arcname=_WEIGHTS_NAME)
         archive.add(os.path.join(serialization_dir, "vocabulary"), arcname="vocabulary")
 
+        if include_in_archive is not None:
+            for archival_target in include_in_archive:
+                archival_target_path = os.path.join(serialization_dir, archival_target)
+                if os.path.exists(archival_target_path):
+                    archive.add(archival_target_path, arcname=archival_target)
```

Comment:
I think it would be good to allow wildcard paths as well. You could use the `glob` module from the standard library.